src/fpylll/gso.pyx

```diff
@@ -13,6 +13,11 @@ from util cimport preprocess_indices, check_float_type
 from fpylll cimport mpz_double, mpz_ld, mpz_dpe, mpz_mpfr, mpz_dd, mpz_qd
 from fplll cimport FT_DOUBLE, FT_LONG_DOUBLE, FT_DD, FT_QD, FT_DPE, FT_MPFR, FloatType
 
+import cython
+import numpy as np
```

Comment:
I'd be happy to get rid of numpy here, if there is any other way that allows to create a pointer double* out of any array-like structure of Python. I followed this tutorial :
https://github.com/cython/cython/wiki/tutorials-NumpyPointerToC

If you have any other way of doing it, I'd be happy to adjust.

But indeed, I would like to avoid looping in python.